examples/platform/silabs/provision/ProvisionStorageDefault.cpp

```diff
@@ -665,12 +665,38 @@ CHIP_ERROR Storage::GetProvisionRequest(bool & value)
 #ifdef SL_MATTER_ENABLE_OTA_ENCRYPTION
 CHIP_ERROR Storage::SetOtaTlvEncryptionKey(const ByteSpan & value)
 {
+#if defined(SL_MBEDTLS_USE_TINYCRYPT)
+    // Tinycrypt doesn't support the key ID, so we need to store the key as a binary blob
+    return SilabsConfig::WriteConfigValueBin(SilabsConfig::kOtaTlvEncryption_KeyId, value.data(), value.size());
```

Comment:
So the key is stored in plain text?

Reply:
Yes, for mbedTLS all keys are stored in plain text, holds true for DAC key as well.